wandb/cli/cli.py

```diff
@@ -936,6 +940,20 @@ def _user_args_to_dict(arguments, argument_type="P"):
     help="Name of the experiment under which to launch the run. If not "
     "specified, 'experiment-id' option will be used to launch run.",
 )
+@click.option(
+    "--entity",
+    "-e",
+    metavar="TEXT",
```

Comment:
can you add valid metavar https://click.palletsprojects.com/en/5.x/documentation/